bpdm-common/src/main/kotlin/org/eclipse/tractusx/bpdm/common/dto/IBasePostalAddressDto.kt

```diff
@@ -19,22 +19,20 @@
 
 package org.eclipse.tractusx.bpdm.common.dto
 
-import com.neovisionaries.i18n.CountryCode
 import io.swagger.v3.oas.annotations.media.Schema
 import org.eclipse.tractusx.bpdm.common.dto.openapidescription.PostalAddressDescription
 
-data class BasePostalAddressDto(
+/**
+ * Contains the common fields for all PostalAddress-DTOs, whether Physical or Alternative respectively Pool or Gate.
+ */
+interface IBasePostalAddressDto {
```

Comment:
Should we really include this interface? It makes sense from a technical perspective as it contains common fields across all addresses but the fields feel just so... random. As a concept for a base postal address the interface really is not very expressive. I feel those three fields could also be added as duplicates in subclasses/subinterfaces as such an interface could feel more confusing. I'm open for discussion though

Reply:
I thought exactly the same. Its only a technical Interface to group some fields without any unique context.
To remove it would also decrease the dept of the address hierarchy

Reply:
When we remove the interface IBasePostalAddressDto perhaps it would make sense to create an IAlternativePostalAddressDto interface.

Reply:
That's not a bad idea, IAlternativePostalAddressDto seems more helpful than IBasePostalAddressDto!

It still seems a little random because it's missing e.g. country that is essential to any address. But this fields unfortunately can't be included because it has a different type in the verbose and normal versions.

Reply:
As decided together we now have only IBasePhysicalPostalAddressDto and IBaseAlternativePostalAddressDto as interfaces.